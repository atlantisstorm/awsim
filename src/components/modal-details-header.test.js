import React from 'react';
import { mount } from 'enzyme';
import ModalDetailsHeader from './modal-details-header';

describe('App component', () => {
  it('starts with a count of 0', () => {
    const title = "fishbone";

    const component = mount(
      <table>
          <tbody>
            <ModalDetailsHeader title={title}/>
          </tbody>
      </table>
    );
    expect(component).toMatchSnapshot();
    const text = component.find('td').text();
    expect(text).toEqual(title);
  });
});