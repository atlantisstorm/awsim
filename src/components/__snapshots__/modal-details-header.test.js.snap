// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component starts with a count of 0 1`] = `
<table>
  <tbody>
    <ModalDetailsHeader
      title="fishbone"
    >
      <tr>
        <td
          className="header"
          colSpan="2"
        >
          fishbone
        </td>
      </tr>
    </ModalDetailsHeader>
  </tbody>
</table>
`;
